### PR TITLE
update wrong page size of test case numa_consistency 

### PR DESCRIPTION
**Pull request by MiriamDeng**
Signed-off-by: Miriam Deng mdeng@redhat.com
ID: 1404109

### CODE REVIEW

File: qemu/tests/numa_consistency.py [hunk 1]
```diff
@@ -91,9 +91,9 @@ def get_vcpu_used_node(numa_node_info, vcpu_thread):
         memory_status, _ = utils_test.qemu.get_numa_status(host_numa_node,
                                                            qemu_pid)
         memory_used_after = memory_status[node_used_host_index]
-
-        memory_allocated = (memory_used_after - memory_used_before) * 4 / 1024
-
+        pa = int(params.get("page_size"))
```
Comment (luckyh) [inline qemu/tests/numa_consistency.py]: I prefer to get this value via python api
https://docs.python.org/2/library/resource.html#resource.getpagesize

**Comment by MiriamDeng** [comment]
@luckyh ,thanks for your suggestion.Since I have updated it,could you please have a look ?

File: qemu/tests/numa_consistency.py [hunk 2]
```diff
@@ -91,9 +92,9 @@ def get_vcpu_used_node(numa_node_info, vcpu_thread):
         memory_status, _ = utils_test.qemu.get_numa_status(host_numa_node,
                                                            qemu_pid)
         memory_used_after = memory_status[node_used_host_index]
-
-        memory_allocated = (memory_used_after - memory_used_before) * 4 / 1024
-
+        pa = resource.getpagesize() / 1024
+        memory_allocated = (memory_used_after -
+                            memory_used_before) * int(pa) / 1024
```
Comment (luckyh) [inline qemu/tests/numa_consistency.py]: `resource.getpagesize() / 1024` will be a integer value, so why `int(pa)`?

Comment (MiriamDeng) [inline qemu/tests/numa_consistency.py]: @luckyh,
  Thanks and I updated it just now,can you help to review it again ？

**Comment by luckyh** [comment]
ACK

File: qemu/tests/numa_consistency.py [hunk 3]
```diff
@@ -91,9 +92,9 @@ def get_vcpu_used_node(numa_node_info, vcpu_thread):
         memory_status, _ = utils_test.qemu.get_numa_status(host_numa_node,
                                                            qemu_pid)
         memory_used_after = memory_status[node_used_host_index]
-
-        memory_allocated = (memory_used_after - memory_used_before) * 4 / 1024
-
+        pa = resource.getpagesize() / 1024
```
Comment (suqinhuang) [inline qemu/tests/numa_consistency.py]: it's better to use a name more clear to instead of pa

**Comment by suqinhuang** [comment]
if the issue is not ppc only, it's better to drop ppc

**Comment by MiriamDeng** [comment]
@suqinhuang ,thank for suggestions and use an explicit name instead at this time.Could you please have a look ? Thanks.

**Comment by suqinhuang** [comment]
you need to update the summary in the commit as well

**Comment by MiriamDeng** [comment]
@suqinhuang ,actually I have updated it at last time please help to check,thanks a lot.

**Comment by MiriamDeng** [comment]
@suqinhuang ,actually I have updated it,can you help to check it ?

**Comment by suqinhuang** [comment]
ack

**Comment by wangxianxian** [comment]
Acked-by: Xianxian Wang xianwang@redhat.com
 
After add this patch, re test case numa.numa_consistency five times,  pass 4 times,  failed 1 time, but the reason for failing is  another issue. Fail debug log is as follows:

04:45:47 ERROR| FAIL 1-smp_16.65536m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.RHEL.7.3.ppc64le.io-github-autotest-qemu.numa.numa_consistency -> TestFail: Expect malloc 256M memory in node 17,but only malloc 239M
Please check more details of the numa node: available: 4 nodes (0-1,16-17)

**Pull request merged**